### PR TITLE
Code Style fixing and small code refactoring

### PR DESCRIPTION
**Pull request by mkoltsov**

### CODE REVIEW

**Comment by corneadoug** [comment]
I guess this PR was rebase with new changes on Master

File: zeppelin-web/app/scripts/app.js [hunk 1]
```diff
@@ -100,9 +97,9 @@ angular
     'puElasticInput',
     'xeditable'
   ])
-  .filter('breakFilter', function() {
+  .filter('breakFilter', function () {
```
Comment (corneadoug) [inline zeppelin-web/app/scripts/app.js]: Actually there is no space here.
Regarding the code style, we are using https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml,
or at least try to fix everything to match it (sometimes its hard to catch them all).

Comment (mkoltsov) [inline zeppelin-web/app/scripts/app.js]: Than you for clarifying

**Comment by Leemoonsoo** [comment]
Thanks for the contribution.
Looks good to me.

**Comment by corneadoug** [comment]
@mkoltsov are you pretty much finished?

**Comment by mkoltsov** [comment]
Yes

**Comment by mkoltsov** [comment]
If everything is fine, can this change be merged so I could submit more severe contribution?

**Pull request closed without merging**